test/avocado/seleniumlib.py

```diff
@@ -95,6 +96,14 @@ def tearDown(self):
         except Exception as e:
             raise Exception('ERR: Unable to close WEBdriver', str(e))
 
+    def debug_logs(self, logs=['browser','driver','client','server']):
```

Comment:
Maybe rename to get_debug_logs or gather_debug_logs?
Otherwise it sounds a bit like a function that debugs.